### PR TITLE
Fix some stuff

### PR DESCRIPTION
**Pull request by ohlookitsAugust**
__**Fixes**__:
* Use `(async() => {})()` instead of `async function run()`
* Requiring `spawn` with deconstructing
* Use ES6 functions
* Deleted constant `OUTPUT` and replaced it with `data` when you defined it with the callback.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
+  console.info('Started the process.');
+  child.stdout.on('data', (data) => {
+    console.log(data);
+    if (PAST > 10) PAST.pop();
+    PAST.push(data);
     
-    if (PAST_STDOUTS.filter(pv => pv === output).length > 5) {
-      console.log("Loop detected -- stopping program!")
+    if (PAST.filter(pv => pv === data).length > 5) {
+      console.info("Loop has been detected -- stopping program...");
```
Comment (zBlakee) [inline src/main.js]: I don't like the wording of this, I prefer my version
This will likely change in the future regardless.

File: src/main.js [hunk 2]
```diff
@@ -1,28 +1,26 @@
...
-
-run();
-async function run() {
-  let child = spawn('node', [process.argv[2]]);
-  let PAST_STDOUTS = []
+const { spawn } = require('child_process');
 
+(async() => {
+  const child = spawn('node', [procress.argv[2]);
+  let PAST = [];
```
Comment (zBlakee) [inline src/main.js]: why have you changed the name of my variable

File: src/main.js [hunk 3]
```diff
@@ -1,28 +1,26 @@
...
   child.unref();
-  child.stdout.setEncoding('utf8')
-  console.log("Started!")
-  child.stdout.on('data', function(data) {
-    console.log(data)
-    if (PAST_STDOUTS > 10) PAST_STDOUTS.pop()
-    const output = data
-    PAST_STDOUTS.push(output)
+  child.stdout.setEncoding('utf8');
+  console.info('Started the process.');
```
Comment (zBlakee) [inline src/main.js]: August stop changing my strings

**Comment by missingbinaries** [comment]
oof

**Comment by ohlookitsAugust** [comment]
void pls

File: src/main.js (hunk 1 above)
Comment (ohlookitsAugust) [reply src/main.js]: Ok will revert.

File: src/main.js (hunk 3 above)
Comment (ohlookitsAugust) [reply src/main.js]: ok

File: src/main.js (hunk 2 above)
Comment (ohlookitsAugust) [reply src/main.js]: Oh, so you wanted to keep `PAST_OUTPUTS`? ok

**Comment by missingbinaries** [comment]
what

**Comment by ohlookitsAugust** [comment]
heck i deleted the repo, so this will have to be closed :eyes:

**Pull request closed without merging**